quactography/hamiltonian/hamiltonian_qubit_edge.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, graph, alpha):
         self.alpha_d = 8 * self.alpha
         self.alpha_f = 8 * self.alpha
         self.alpha_i = self.alpha
+        self.alphai = alpha
```

Comment:
I think this constructor should have a docstring to help the reader know what are the expect type and possibly a range for alpha (is it -1 to 1, 0 to infinity, is it -infinity to 0?)

Also I think variable name could be clearer, for example self.alpha_d vs self.alpha_f vs self.alpha_i (what does d/f/i stand for?)

Or something like:
`self.alpha = alpha * self.graph.all_weights_sum / graph.number_of_edges`
could be 
`self.alpha_normalized = alpha * self.graph.all_weights_sum / graph.number_of_edges`
(because having multiple alpha variable without context is very confusing later in the code

Reply:
or at least some comments to explain what the variables are